homework-1/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+public class BankAccount
+{
+    private int balance;
+
+    public BankAccount()
+    {
+        balance = 0;
+    }
+
+    public void Deposit(int amount)
+    {
+        balance += amount;
+        Console.WriteLine($"Deposited Rs{amount}. New balance: Rs{balance}");
+    }
+
+    public void Withdraw(int amount)
+    {
+        
+        if (balance >= amount)
+        {
+            balance -= amount;
+            Console.WriteLine($"Withdrawn Rs{amount}. New balance: Rs{balance}");
+        }
+        else
+        {
+            Console.WriteLine("Error: Insufficient balance.");
+        }

```

Comment:
This is correct but what you can do to make it more readable is to always have the negative conditions first.